source/MRMesh/MRVoxelsLoad.cpp

```diff
@@ -614,13 +615,20 @@ tl::expected<SimpleVolume, std::string> loadRaw( const std::filesystem::path& pa
             return tl::make_unexpected( "Cannot parse filename: " + filename );
         auto zvString = filename.substr( yvEndChar + 1, zvEndChar - ( yvEndChar + 1 ) );
         outParams.voxelSize.z = float( std::atof( zvString.c_str() ) / 1000 ); // convert mm to meters
+
+        auto gtEndChar = filename.find( "_", zvEndChar + 1 );
+        if ( gtEndChar != std::string::npos )
+        {
+            auto gtString = filename.substr( zvEndChar + 1, gtEndChar - ( zvEndChar + 1 ) );
+            outParams.gridLevelSet = gtString == "1"; // convert mm to meters
+        }
     }
```

Comment:
will it work with old files without `G` prefix?

Reply:
yes, it will be